apps/dashboard/app/(app)/ratelimits/[namespaceId]/overrides/page.tsx

```
@@ -49,37 +54,67 @@ export default async function OverridePage(props: Props) {
+        </Navbar.Actions>
+      </Navbar>
+      <PageContent>
+        <SubMenu navigation={navigation(props.params.namespaceId)} segment="overrides" />
 
-      <Separator className="my-8" />
+        <div className="flex flex-col gap-8 mt-8">
+          <PageHeader
+            className="m-0"
+            title="Overridden identifiers"
+            actions={[
+              <Badge variant="secondary" className="h-8">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)